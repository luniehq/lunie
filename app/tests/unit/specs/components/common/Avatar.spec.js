import { shallowMount } from "@vue/test-utils"
import Avatar from "common/Avatar"

describe(`Avatar`, () => {
  it("should show an avatar", () => {
    const wrapper = shallowMount(Avatar, {
      propsData: {
        address: "cosmos1234",
      },
    })
    expect(wrapper.element).toMatchSnapshot()
  })
})
