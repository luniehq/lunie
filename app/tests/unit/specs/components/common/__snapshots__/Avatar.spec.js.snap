// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Avatar should show an avatar 1`] = `
<div>
  <svg
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 50 50"
    xmlns="http://www.w3.org/2000/svg"
  >
    <path
      d="M25 1L25 13L19 13ZM37 13L25 13L25 7ZM25 49L25 37L31 37ZM13 37L25 37L25 43ZM13 13L13 25L7 25ZM49 25L37 25L37 19ZM37 37L37 25L43 25ZM1 25L13 25L13 31ZM13 13L25 13L25 14.9L20 25L13 25ZM37 13L37 25L35.1 25L25 20L25 13ZM37 37L25 37L25 35.1L30 25L37 25ZM13 37L13 25L14.9 25L25 30L25 37Z"
      fill="#59c794"
    />
    <path
      d="M1 1L13 1L13 7ZM49 1L49 13L43 13ZM49 49L37 49L37 43ZM1 49L1 37L7 37Z"
      fill="#ace3c9"
    />
  </svg>
</div>
`;
